src/modules/data/sound.c

```diff
@@ -25,13 +25,15 @@ struct Sound {
   uint32_t sampleRate;
   uint32_t frames;
   uint32_t cursor;
+  void *callbackMemo;
+  SoundDestroyCallback callbackMemoDestroy;
```

Comment:
In the spatializer, I named the custom data field "spatializerMemo", so I named it "memo" here as well.